docs/source/author/chapters.rst

```diff
@@ -33,7 +31,7 @@ The ``content`` directory for the chapters application contains
 Content Files
 ==============================================================================
 
-There are 3 different types of files used for adding content to CS Unplugged:
+There are 3 different types of files used for adding content to the CS Field Guide:
 
 - Content Markdown files
 - YAML files containing translatable strings
```

Comment:
These yaml files don't exist, I think they're .po files but those shouldn't be edited so it shouldn't be mentioned here anyway